src/aiida_quantumespresso/parsers/xspectra.py

```diff
@@ -87,6 +87,28 @@ def parse(self, **kwargs):
         xy_data.set_y(y_arrays, y_names, y_units)
 
         parsed_data, logs = parse_stdout_xspectra(filecontent=out_file, codename='XSpectra')
+
+        # Parse some additional info which the stdout does not reliably report
+        input_params = self.node.get_incoming().get_node_by_label('parameters').get_dict()
+
+        epsilon_vector = [
+            float(input_params['INPUT_XSPECTRA'][f'xepsilon({n})']) for n in [1, 2, 3]
+        ]
+        parsed_data['epsilon_vector'] = epsilon_vector
+
+        if 'xcoordcrys' in input_params['INPUT_XSPECTRA']:
+            if input_params['INPUT_XSPECTRA']['xcoordcrys']:
+                parsed_data['vector_coord_system'] = 'crystal'
+            else:
+                parsed_data['vector_coord_system'] = 'cartesian'
+        else:
+            parsed_data['vector_coord_system'] = 'crystal'
```

Comment:
This logic seems a bit weird to me, is it correct? Basically `vector_coord_system` is getting set to `crystal` if `xcoordcrys` is not set *or* `xcoordcrys` is "truthy" (i.e. not `None`, empty string etc.). Otherwise it is `cartesian`.

Reply:
I see in the input doc that this is a boolean. Why convert to a string? Why not simply do
```python
parsed_data['xcoordcrys'] = parameters.get('xcoordcrys', True)
```

Actually, now that I read all this. These are all input parameters defined by the user. Why do we "parse" them and add them to the output parameters? If they are already in the input, why not get them from there if needed?